pkg/api/validation/validation_test.go

```diff
@@ -1440,7 +1469,7 @@ func TestValidateService(t *testing.T) {
 		{
 			name: "too long generateName",
 			tweakSvc: func(s *api.Service) {
-				s.GenerateName = strings.Repeat("a", 25)
+				s.GenerateName = strings.Repeat("a", 65)
```

Comment:
Not pushed yet?

On Tue, May 19, 2015 at 10:49 AM, Vish Kannan <notifications@github.com>
wrote:

> In pkg/api/validation/validation_test.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8140#discussion_r30626178>
> :
>
> > @@ -1440,7 +1469,7 @@ func TestValidateService(t *testing.T) {
> >  		{
> >  			name: "too long generateName",
> >  			tweakSvc: func(s *api.Service) {
> > -				s.GenerateName = strings.Repeat("a", 25)
> > +				s.GenerateName = strings.Repeat("a", 65)
>
> Done
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/8140/files#r30626178>
> .
>